src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@using Esfa.Recruit.Shared.Web.Mappers
+@using Esfa.Recruit.Vacancies.Client.Domain.Entities
+@model Esfa.Recruit.Employer.Web.ViewModels.Part1.Duration.DurationViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <a asp-show="@Model.PageInfo.IsWizard" asp-route="@RouteNames.Dates_Get" esfa-automation="link-back" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Back</a>
+
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames }'/>
+
+        <partial name="@PartialNames.ReviewSummary" for="Review"/>
+
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">Duration and working hours</h1>
+
+        <form asp-route="@RouteNames.Duration_Post" asp-route-wizard="@Model.PageInfo.IsWizard" novalidate>
+            <div esfa-validation-marker-for="Duration" class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                    <label asp-for="Duration" class="govuk-label govuk-!-font-weight-bold">
+                        How long is the whole apprenticeship, including work and training?
+                        <span class="govuk-visually-hidden"> (enter a numeric value and then select Month(s) or Year(s) in the next field)</span>
```

Comment:
Are we doing this for all the inputs?

Reply:
I don't know. This was just a cut-n-paste job.